### DISCUSSION TITLE
Huawei Solar Integration (German Language) - no Values delivered for sensor.power_inverter_1_input in Huawei Solar - PEES

### DISCUSSION THREAD

**Discussion by ulkr1** [JensenNick/huawei_solar_pees#89] [Forum]
Hi,
I "installed" Huawei Solar - PEES as described, 
- installed Huawei Solar integration by wlcr https://github.com/wlcrs/huawei_solar / as I am german, the sensors got german sensor names
- installed Energi Data Service integration by MTrab
- did all the dir creation / copy/paste of the two package files
- addes the entry in the configuration.yaml
- as described, renamed the four sensors

----------------------------------
BTW:
There is a mismatch in the various descriptions

At https://github.com/JensenNick/huawei_solar_pees 3.2 decription says 

inverter_input_power / inverter_input_power_2 (if you have two inverters), power_meter_active_power and battery_charge_discharge_power. 

while a bit later it says
# - 'sensor.inverter_input_power' (from the Huawei Solar integration)
# - 'sensor.power_inverter_2_input' (from the Huawei Solar integration)
# - 'sensor.power_meter_active_power' (from the Huawei Solar integration)
# - 'sensor.battery_charge_discharge_power' (from the Huawei Solar integration)

> diff between 2. sensor - inverter_input_power_2 / power_inverter_2_input  

In file huawei_solar_pees.yaml it is correct:
# Make sure the naming of your power input sensors is correct.
# - 'sensor.inverter_input_power' (from the Huawei Solar integration)
# - 'sensor.inverter_input_power_2' (from the Huawei Solar integration)
# - 'sensor.power_meter_active_power' (from the Huawei Solar integration)
# - 'sensor.batteries_charge_discharge_power' (from the Huawei Solar integration)
-----------------------------------

So I renamed 
sensor.inverter_input_power > sensor.inverter_eingangsleistung
sensor.inverter_input_power_2 > sensor.inverter_eingangsleistung_2
sensor.power_meter_active_power > sensor.power_meter_wirkleistung
sensor.battery_charge_discharge_power > sensor.batteries_lade_entladeleistung

The mapping is from the language files (en / de) of installed Huawei Solar integration by wlcr

Now values are delivered for many Huawei Solar PEES sensors, but not for sensor.power_inverter_1_input which should be equal or similar to sensor.inverter_input_power(en) / sensor.inverter_eingangsleistung(de). The value of sensor.inverter_input_power_2 is delivered and equal to sensor.power_inverter_2_input (en) / sensor.inverter_eingangsleistung_2 (de).

Any chance to get the sensor.power_inverter_1_input value?

Regards,
Ulrich

**Comment by JensenNick** [comment]
The 'sensor.power_inverter_1_input' is automatically derived from 'sensor.inverter_input_power', so there must be something wrong with your renaming.

**Comment by ulkr1** [comment]
Tnx, got it, now it works.

Thank you for the great work,

Ulrich

 

Von: JensenNick ***@***.***> 
Gesendet: Mittwoch, 6. November 2024 20:36
An: JensenNick/huawei_solar_pees ***@***.***>
Cc: ulkr1 ***@***.***>; Author ***@***.***>
Betreff: Re: [JensenNick/huawei_solar_pees] Huawei Solar Integration (German Language) - no Values delivered for sensor.power_inverter_1_input in Huawei Solar - PEES (Discussion #89)

 

The 'sensor.power_inverter_1_input' is automatically derived from 'sensor.inverter_input_power', so there must be something wrong with your renaming.
